src/main/java/seedu/address/logic/commands/BatchCommand.java

```diff
@@ -84,8 +85,8 @@ public boolean equals(Object other) {
 
```

Comment:
When i was compiling, intellij notifies that this part has raw use of parameterised class. But if there's no problem i think can still leave it as it is

Reply:
I added the suppress warning tag.